### PR TITLE
Update Codacy badge to use legacy organization (for now) (#107)

### PR DESCRIPTION
**Pull request by ssciolla**
This PR changes the Codacy badge in the README so the organization in use matches the legacy organization in use by our other projects. The PR aims to resolve issue #107.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # course-inventory
 
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/431390d7c73d47cfbf9654f0182ba6d6)](https://www.codacy.com/gh/tl-its-umich-edu/course-inventory)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/299102a0330d4dfd8e9f8d5ef36362b5)](https://www.codacy.com/manual/ITS_Teaching_And_Learning/course-inventory?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=tl-its-umich-edu/course-inventory&amp;utm_campaign=Badge_Grade)
```
Comment (pushyamig) [inline README.md:4]: What are all these params you are adding after the Question mark? can you refer to the documentation that made you chose those options?

Comment (ssciolla) [reply README.md:4]: Sorry, in Codacy, on the repository page under the General tab, there is a place where you can copy the Codacy Badge in Markdown. I got this from there.

Comment (ssciolla) [reply README.md:4]: I could remove the parameters if you want?

Comment (pushyamig) [reply README.md:4]: That's what codacy seems to suggest then you shouldn't be removing it.

Comment (ssciolla) [reply README.md:4]: Based on what @lsloan had before, they probably aren't necessary and are primarily used for Codacy's own analytics. Not entirely sure though.

Comment (pushyamig) [reply README.md:4]: I am ok either way, Looking at this [article](https://support.codacy.com/hc/en-us/articles/212799365-Badges) they are saying if you want to track code performance for the different branch you can use param for doing that. I don't feel the need to dig deep-down what all those params mean. I got curious about what those are and purpose

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
```
Comment (lsloan) [inline README.md:3]: As discussed earlier, I don't think the params on the link URL are necessary, but maybe they're useful for analytics.

**Pull request merged**